setup.py

```diff
@@ -13,6 +13,18 @@
     use_setuptools()
     from setuptools import setup, find_packages
 
+install_requires = [
+    'wagtail>=4.1',
```

Comment:
is there some reason this was added? This should in theory work in any wagtail version (or even non wagtail projects really)

Reply:
It's just that Wagtail versions before 4.1 have reached their end of life: https://endoflife.date/wagtail